### DISCUSSION TITLE
How to install lunarvim with neovim binary?

### DISCUSSION THREAD

**Discussion by sourproton** [LunarVim/LunarVim#4294] [Q&A] [answered]
Due to permission issues, my neovim installation isn't done from the package manager, but from downloading and aliasing the neovim binary.

When running the lunarvim install command, I get the error

```
Detecting platform for managing any additional neovim dependencies
[ERROR]: Unable to find dependency [neovim]
Please install it first and re-run the installer. Try: sudo dnf install -y neovim
```

Under these conditions, how can I install lunarvim ?

**Comment by LostInTheLogs** [comment, marked as answer]
[answered on reddit by u/theallwaystnt](https://www.reddit.com/r/lunarvim/comments/14xkb3f/comment/jrp74d2/?utm_source=share&utm_medium=web2x&context=3):
> I'm fairly certain the directory that contains the nvim binary just needs to be added to your PATH before installing lunarvim.
